db/system_keyspace.cc

```diff
@@ -624,6 +624,30 @@ schema_ptr system_keyspace::large_cells() {
     return large_cells;
 }
 
+schema_ptr system_keyspace::large_data() {
+    static thread_local auto large_data = [] {
+        auto id = generate_legacy_id(NAME, LARGE_DATA);
+        return schema_builder(NAME, LARGE_DATA, id)
+                .with_column("keyspace_name", utf8_type, column_kind::partition_key)
+                .with_column("table_name", utf8_type, column_kind::partition_key)
+                .with_column("sstable_name", utf8_type, column_kind::partition_key)
+                // We want the larger ratios first, so use reversed_type_impl
+                .with_column("ratio", reversed_type_impl::get_instance(float_type), column_kind::clustering_key)
+                .with_column("metric", utf8_type, column_kind::clustering_key)
+                .with_column("partition_key", utf8_type, column_kind::clustering_key)
+                .with_column("clustering_key", utf8_type, column_kind::clustering_key)
+                .with_column("column_name", utf8_type, column_kind::clustering_key)
+                .with_column("value", long_type)
+                .with_column("compaction_time", timestamp_type)
+                .set_comment("data size threshold violations")
+                .with_version(generate_schema_version(id))
+                .set_gc_grace_seconds(0)
+                .set_caching_options(caching_options::get_disabled_caching_options())
+                .build();
+    }();
+    return large_data;
+}
```

Comment:
I thought you aren't doing this.

Reply:
I didn't really like the idea of simply adding a "row_count" field to large_partitions because it would be a second-class citizen (the output would still be sorted according to "partition_size").
So went for a more general thing.

Reply:
Doesn't it break compatibility?